front-end/src/components/SidebarList/SidebarList.jsx

```diff
@@ -78,9 +77,8 @@ export const SidebarList = () => {
           )}
         </ListItem>
         {currentUser?.role === 'student' && (
-          <ListItem className={styles.sidebarItem}>
+          <ListItem>
             <Link to='/app/subscriptions' className={styles.sidebarLink}>
-              <FontAwesomeIcon className={styles.sidebarIcon} icon={faBell} />

```

Comment:
Are you sure that you don't need icon?

Reply:
I maybe accidently removed icon and styles